src/components/App.jsx

```diff
@@ -7,10 +7,10 @@ import classNames from 'classnames'
 import Sidebar from './Sidebar'
 import Alerter from './Alerter'
 
-const App = ({ t, children }) => (
+export const App = ({ t, children }) => (
   <div className={classNames(styles['pho-wrapper'], styles['coz-sticky'])}>
 
-    <Sidebar />
+    <Sidebar t={t} />
```

Comment:
Indeed, it's facilitate the tests but even that, after thinking, I think this way is also better because we shouldn't provide translate function at each component. We should follow the 'Single source of truth' concept, even if it's not a ressource here. Each parent should be provide the translate function as a prop if the child component need it. It's more testable and more maintainable as well (only one import of the I18n lib).

But I am opened to discussion :)